doc/nrf/links.txt

```diff
@@ -95,6 +93,18 @@
 .. _`Project CHIP Protocol Overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/README.md
 .. _`nRF Connect platform overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/a008cb/docs/guides/nrfconnect_platform_overview.md
 
+.. _`Azure support for Asset Tracker v2`: https://github.com/nrfconnect/sdk-nrf/pull/3950
+
+.. ### Source: github.io
+
+.. _`TinyCBOR`: https://intel.github.io/tinycbor/current/
+
+.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.5.x/docs/device-cloud-protocol/FourKindsOfData.html#timestamping
```

Comment:
```suggestion
.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.5.x/docs/guides/FourKindsOfData.html#timestamping
```

Reply:
I don't want to touch the link URLs in this PR, just their placement within the file.
JIRA created: https://projecttools.nordicsemi.no/jira/browse/NCSDK-9285